### DISCUSSION TITLE
ipsec.conf support

### DISCUSSION THREAD

**Discussion by mike-mcmaster** [strongswan/strongswan#1743] [General]
Hi,

I have a system that still uses ipsec.conf, I have read that it has been deprecated, but it still works - thankfully!
We are planning to convert to swanctl.conf soon.

Just so I understand the timeline and risk, if or when are you planning to remove ipsec.conf completely.
I don't want the application to suddenly break in the meantime,

Many thanks,
-Mike.

**Comment by tobiasbrunner** [comment]
There is currently no timeline for its complete removal. But with the upcoming 6.0 release the _stroke_ plugin (and starter/ipsec etc.) will be disabled, by default (see [here](https://docs.strongswan.org/docs/6.0/news/whatsNew.html)). I guess distros like Debian will probably still enable it.
